upload_utility/config.py

```diff
@@ -0,0 +1,27 @@
+NETWORKS = {
```

Comment:
do we still nee network_id in config?

Reply:
Yes, lambda exception handling need networkID, and also We are planning to have infrastructure layer to store user history so i kept it this way in initial commit itself.